src/shared/components/Checkbox/Checkbox.styles.tsx

```diff
@@ -16,8 +15,8 @@ export const CheckboxLabel = styled.label<CheckboxLabelProps>`
 `
 
 export const LabelText = styled(Text)`
-  margin-left: 10px;
-  color: ${colors.gray[400]};
+  color: ${colors.gray[50]};
+  line-height: ${sizes(7)};
 `
 
 export const Container = styled.div<CheckboxStateProps>`
```

Comment:
Cannot comment on the line I want to, but the negative margin to the left should be now only `sizes(1)`: 
<img width="549" alt="Screenshot 2021-04-08 at 11 26 19" src="https://user-images.githubusercontent.com/12646744/114002773-778c5c80-985d-11eb-93df-8590f87c9ef4.png">